### ISSUE TITLE
[DB] Problem with database tables when saved as resources

### ISSUE THREAD

**Issue by ababen1** [Kehom/GodotAddonPack#37]
Ok, so when you create a table you have 2 options: save it as "embedded" or save it as a resource in your db's folder
so i used the latter and created a table. then i removed the table i created with the little minus icon here:
![image](https://user-images.githubusercontent.com/80917990/135490405-0ed24c96-f89a-4e30-9fc9-b8b278c7501e.png)
and when i added the same table back (by dragging it from the filesystem) it showed up in the editor but when i ran the code and printed the list of tables it wasnt there. 
so basically the bug is that if you save a table as a resource it stops working if you remove it and add it back again
the embedded ones work fine though so there's a workaround
(btw i really appreciate your addons ty 👉 👈 )

**Comment by Kehom** [comment]
Oh! Thank you for notifying about this issue.

I completely forgot to save the database when a table is dropped into it!

Pushing the fix.
